README.md

```diff
@@ -236,3 +236,4 @@ void app.free(reaction r);
 ### Examples
 
 - [`Complex`](examples/complex.cpp): Demonstrates several different reaction types for controlling a servo, monitoring inputs and blinking an LED.
+- [`Method_cb`](examples/method_cb.cpp): Demonstrates how to use class methods as callbacks.
```

Comment:
Could we change this to `Handlers` or `Callbacks` - and just turn it into examples of different  handler options?

1. Simple function
2. Lambda function
3. Simple function wrapping class method
4. Lambda function wrapping static class method
5. Non example of local captures

Or do you think it would be best to split these into separate examples?

I think we probably need both though, then we could maybe have `Callbacks` and `Advanced Callbacks`.